Core/InternalData.lua

```diff
@@ -56,8 +56,14 @@ I.Data.Contributor = {
 
     [I.Enum.Developers.WOLEN] = {
       [I.Enum.Flavor.RETAIL] = {
-        -- yes
-        ["Wolen-SomeRealm"] = true,
+        -- Mal'Ganis
+        ["Wolen-Mal'Ganis"] = true,
```

Comment:
Don't think `'` is valid, can you confirm these are the exact realm names with `/tx export names` ?